src/Components/ADT3DSceneBuilder/Internal/Behaviors/Internal/WidgetsTab.tsx

```diff
@@ -201,6 +201,8 @@ function getListItems(
                 return widget?.widgetConfiguration?.displayName;
             case WidgetType.DataHistory:
                 return widget?.widgetConfiguration?.displayName;
+            case WidgetType.PowerBI:
+                return widget?.widgetConfiguration?.label;
```

Comment:
any reason we called it label instead of displayName like we did with the rest of them?